src/helpers/is-prod.ts

```diff
@@ -0,0 +1,3 @@
+export function isProd() {
+  return process.env.NODE_ENV === "production";
+}
```

Comment:
can you rename `isProd` to `isProduction`? We avoid abbreviations

Reply:
[addressed here](https://github.com/probot/probot/pull/1598/commits/a04d4c6526073aac59d496d997748f819cabbd65)